src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -782,17 +778,23 @@ private void ThrowFailedToContactAllNodes<TResult>(RavenCommand<TResult> command
             var firstException = command.FailedNodes.Values.First();
             
             // this means that all exception has the same type
-            if (command.FailedNodes.All(x=>x.Value.GetType() == firstException.GetType() && x.Value.Message == firstException.Message))
+            if (command.FailedNodes.All(x=> ExceptionAreEqual(x.Value, firstException)))
```

Comment:
Regardless, we want to get a list of exception and url in the errors.

Reply:
OK, so I'll remove this check.

Reply:
https://github.com/ravendb/ravendb/pull/5055